dts/msm8916-motorola-harpia.dts

```diff
@@ -8,6 +8,12 @@
 / {
 	qcom,msm-id = <206 0>;
 	qcom,board-id = <0x4E 0x81B0>;
+	qcom,board-id = <0x4D 0x8000>;
+	qcom,board-id = <0x4D 0x80A0>;
+	qcom,board-id = <0x4D 0x8100>;
+	qcom,board-id = <0x4D 0x81A0>;
+	qcom,board-id = <0x4D 0x81AD>;
+	qcom,board-id = <0x4D 0x81B0>;
```

Comment:
That's not how it works. Everything you have a `qcom,board-id =` line, you override previous ones. So if anything you would need something like below, everything with comma.

Reply:
So I got lucky with the first one then when I tested? Will change and re-test. :+1:

Reply:
Ok. Fixed this up the way I understood it to work. Flashed it to my XT1607 harpia and seems to work.

Reply:
> fastboot oem lk_log && fastboot get_staged <output file>

```
[160] Updating board hardware id: 0x8 -> 0x4e
[160] Updating board hardware subtype: 0x0 -> 0xb0
```

I found the 81B0 mentioned above in lk_log as well, in

```
[160] Command line from primary bootloader:
...
, system_rev = 0x81b0
...
```